src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        Ball ballone = lottery.getRandomBall();
+        Ball balltwo = lottery.getRandomBall();
+        Ball ballthree = lottery.getRandomBall();
+
+        System.out.println(ballone);
+        System.out.println(balltwo);
+        System.out.println(ballthree);
```

Comment:
Use for loop for creating several objects of the same class. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class).
```suggestion
        for(...) {
             //create a ball and print it
        }
```